gunicorn-minimal.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+gunicorn --bind 0.0.0.0:$PORT --workers=1 --threads=8 src.app:app --timeout=900
```

Comment:
Why two files? why not invoke this file from the other file if this is needed?

Reply:
This is used to run gunicorn locally as current `gunicorn.sh` has dependency on credential manager and jq.

We will resolve it by having single gunicorn file by checking with `FLASK_ENV` environment variable.

cc: @ub0005